lib/coinbase/user.rb

```diff
@@ -62,6 +62,65 @@ def list_wallet_ids
       wallets.data.map(&:id)
     end
 
+    # Saves a wallet to local file system. Wallet saved this way can be re-instantiated with load() function,
+    # provided the backup_file is available.
```

Comment:
Add comment saying this is only for testing purposes, it's insecure, etc.

Reply:
There may also be some language missing about the behavior of what happens when you call save twice (i.e. does it overwrite, or append?)

Reply:
updated. since the function takes wallet_model itself, we dont need to add details on sessions i think